zap/src/main/java/org/parosproxy/paros/core/scanner/PluginStats.java

```diff
@@ -38,11 +42,11 @@
     /**
      * Constructs a {@code PluginStats}.
      *
-     * @param pluginName the name of the plugin.
      * @see #start()
      */
-    PluginStats(String pluginName) {
-        this.pluginName = pluginName == null ? "" : pluginName;
+    public PluginStats(Plugin plugin) {
```

Comment:
Keep package? External code should not need to create instances.

Reply:
Thats weird, I didnt mean to change that :/

Reply:
Fixed.